superset/daos/tag.py

```diff
@@ -257,3 +266,90 @@ def get_tagged_objects_for_tags(
                 for obj in saved_queries
             )
         return results
+
+    @staticmethod
+    def user_favorite_tag(tag_id: int) -> None:
```

Comment:
naming: maybe favor_tag_by_id_for_current_user

Reply:
or you can pass in user_id as param so it will be like `def favor_tag_by_id(tag_id: int, user_id: int) -> None`

Reply:
is there any authorization that you need to check or any user can favor any tag?

Reply:
right now any user can favorite a tag and matches the current functionality for other resources